fpdf/fpdf.py

```diff
@@ -1974,7 +1974,15 @@ def _out(self, s):
         if(self.state == 2):
             self.pages[self.page]["content"] += (s + "\n")
         else:
-            self.buffer += (s + "\n")
+            if PY3K:
+                if not isinstance(s, bytes):
```

Comment:
yes. In my debug/test, i see that some call to _out are str and others are bytes, in PY3, and BytesIO needs bytes.